src/utils/Store.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable max-classes-per-file */
+import EventBus from './EventBus';
+import { set, isEqual } from './helpers';
+import Block from './Block';
+import { User } from '../api/AuthApi';
+import { ChatInfo } from '../api/ChatApi';
+import { Message as MessageInfo } from '../controllers/MessageController';
+
+export enum StoreEvents {
+  Updated = 'updated',
+}
+type State = {
+  user: User;
+  chats: ChatInfo[];
+  messages: Record<number, MessageInfo[]>;
+  selectedChat?: number;
+  createdChat?: number;
+
+}
+
+export class Store extends EventBus {
+  private state: any = {};
```

Comment:
Если есть тип для state, я бы не использовал тут any, а использовал бы этот тип
```suggestion
  private state: State = {};
```

Reply:
Поправила